src/mintpy/objects/euler_pole.py

```diff
@@ -17,9 +17,15 @@
 #  Goudarzi, M. A., Cocard, M. & Santerre, R. (2014), EPC: Matlab software to estimate Euler
 #    pole parameters, GPS Solutions, 18, 153-162, doi: 10.1007/s10291-013-0354-4
 
+import collections
+import os
 
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
 import numpy as np
 import pyproj
+from shapely import geometry
 
 from mintpy.constants import EARTH_RADIUS
 
```

Comment:
I would suggest defining a variable here for the two PMM plate boundary files, such as below, then the other code could just use `PLATE_BOUNDARY_FILE['GSRM']` as the file name, without caring about the detail.

```python
import mintpy

PLATE_BOUNDARY_FILE = {
    'GSRM' : os.path.join(mintpy.__path__, 'data/plate_boundary/GSRM/GSRM_plate_outlines.gmt'),
    'MORVEL56' : os.path.join(mintpy.__path__, 'data/plate_boundary/MORVEL56/All_boundaries'),
}
```

Reply:
sounds good